### PR TITLE
Adding clusterloader env variables

### PR DESCRIPTION
**Pull request by krzysied**
Adding env variables that will be used by clusterloader.

### CODE REVIEW

File: kubetest/e2e.go [hunk 1]
```diff
@@ -697,7 +704,6 @@ func kubemarkTest(testArgs []string, dump string, o options, deploy deployer) er
 			os.Environ(),
 			"KUBERNETES_PROVIDER=kubemark",
 			"KUBE_CONFIG_FILE=config-default.sh",
-			fmt.Sprintf("KUBECONFIG=%s/test/kubemark/resources/kubeconfig.kubemark", cwd),
```
Comment (wojtek-t) [inline kubetest/e2e.go]: Won't this break current tests (those without cluster loader)?

Comment (krzysied) [reply kubetest/e2e.go]: It shouldn't. `KUBECONFIG` is set 4 lines above. I assume `os.Environ()` will read it from env.

File: kubetest/e2e.go [hunk 2]
```diff
@@ -678,6 +678,9 @@ func kubemarkTest(testArgs []string, dump string, o options, deploy deployer) er
 		if err := os.Setenv("KUBE_MASTER_IP", strings.TrimSpace(string(masterIP))); err != nil {
 			return err
 		}
+		if err := os.Setenv("MASTER_IP", strings.TrimSpace(string(masterIP))); err != nil {
```
[the remark this answers is not in the document]
Comment (krzysied) [reply kubetest/e2e.go]: Done.

**Comment by spiffxp** [comment]
/approve

btw I don't see why this needs to hit the root OWNERS file, I'm going to add one for this

**Pull request merged**